architecture-model-generator/modules/architecture-model-generator-ls-extension/src/main/java/io/ballerina/architecturemodelgenerator/extension/persist/PersistERModelRequest.java

```diff
@@ -0,0 +1,37 @@
+/*
+ *  Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
```

Comment:
Apply where necessary.

```suggestion
 *  WSO2 LLC. licenses this file to you under the Apache License,
```